scikeras/__init__.py

```diff
@@ -3,12 +3,12 @@
 __author__ = """Adrian Garcia Badaracco"""
 __version__ = "0.2.1"
 
+from tensorflow import keras
 
-# Monkey patch log_cosh reference
-# See https://github.com/tensorflow/tensorflow/pull/42097
-# Will be removed whenever the
-# min supported version of tf incorporates the fix
-from tensorflow.python import keras  # noqa
+from scikeras import _saving_utils as saving_utils
```

Comment:
Why are you making `saving_utils` a public module? Or, why should the end-user use any function in `saving_utils`?

Reply:
Great point. This was just a blunder on my part, I'll just import it as `_saving_utils` and keep it private.

Reply:
I think you can remove the import line. IIRC, it doesn't change this usage:

```
import scikeras._saving_utils
from scikeras import _saving_utils
```

Reply:
I'm planning on doing:
```python3
from scikeras import _saving_utils
keras.Model.__reduce__ = _saving_utils.pack_keras_model
```
I think that's what you're suggesting but just checking